### PR TITLE
Articles added to home page

### PR DESCRIPTION
**Pull request by Romain-Constant**
Articles added to home page and active link on navbar

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+
```
Comment (fantinerudent) [inline frontend/index.html:10]: tu peux supprimer la ligne vide

File: frontend/src/components/article/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        {newArticle.results
+          ? newArticle.results.map(
+              (article, index) =>
+                index < 10 && (
+                  <ArticleCard
+                    key={article.id}
+                    title={article.title}
+                    url={article.url}
+                    imageUrl={article.image_url}
+                    /* summary={article.summary} */
```
Comment (fantinerudent) [inline frontend/src/components/article/Article.jsx]: comment?

File: frontend/src/components/article/Article.module.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap");
+
+.articlesContainer {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
```
Comment (fantinerudent) [inline frontend/src/components/article/Article.module.css]: delete line

File: frontend/src/components/article/Article.module.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  border-radius: 15px;
+  text-align: center;
+
```
Comment (fantinerudent) [inline frontend/src/components/article/Article.module.css]: delete line

File: frontend/src/components/article/Article.module.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  /* overflow: hidden; */
```
Comment (fantinerudent) [inline frontend/src/components/article/Article.module.css]: delete comment

File: frontend/src/components/articleCard/ArticleCard.module.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+.artImg {
+  max-width: 65%;
+
+  border-radius: 15px;
+}
+
+.artTitle {
+  font-size: 1.6vh;
+  font-weight: 200;
+
```
Comment (fantinerudent) [inline frontend/src/components/articleCard/ArticleCard.module.css]: delete line

File: frontend/src/components/articleCard/ArticleCard.module.css [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  border-radius: 15px;
+  color: whitesmoke;
+  align-items: center;
+  padding: 10px;
+  margin: 10px 20px;
+}
+
+.artImg {
+  max-width: 65%;
+
```
Comment (fantinerudent) [inline frontend/src/components/articleCard/ArticleCard.module.css]: delete line

File: frontend/src/components/footer/Footer.jsx [hunk 8]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { FaGithub } from "react-icons/fa";
+import styles from "./Footer.module.css";
+
+function Footer() {
+  return (
+    <div className={styles.footerContainer}>
+      <p className={styles.footerText}>
+        {" "}
```
Comment (fantinerudent) [inline frontend/src/components/footer/Footer.jsx]: comment

File: frontend/src/components/footer/Footer.module.css [hunk 9]
```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap");
+
+.footerContainer {
+  /*   background-color: rgba(0, 0, 0, 0.398); */
```
Comment (fantinerudent) [inline frontend/src/components/footer/Footer.module.css]: delete comment

File: frontend/src/components/nextLaunch/nextLaunchMobile/NextLaunchMobile.module.css [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+  border-radius: 15px;
+  background-color: rgba(0, 0, 0, 0.535);
+}
+
+.nextLaunchMobileContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
```
Comment (fantinerudent) [inline frontend/src/components/nextLaunch/nextLaunchMobile/NextLaunchMobile.module.css]: delete line

File: frontend/src/pages/Agencies.module.css [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.padsAndArticlesContainer {
+  display: flex;
+
+  /* align-items: stretch; */
```
Comment (fantinerudent) [inline frontend/src/pages/Agencies.module.css]: delete comment & line

File: frontend/src/pages/Agencies.jsx [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+import NavBar from "@components/navBar/NavBar";
+import Article from "../components/article/Article";
+import NavbarMobile from "../components/navbarMobile/NavbarMobile";
+import Footer from "../components/footer/Footer";
+import styles from "./Agencies.module.css";
+
+function Agencies() {
+  return (
+    <div className={styles.agenciesContainer}>
+      <NavbarMobile />
```
Comment (fantinerudent) [inline frontend/src/pages/Agencies.jsx]: why did you import the nav bar in every component instead of in the app.jsx?

**Review by fantinerudent — CHANGES_REQUESTED** [review]
the major problem  I  see is that you import navbar / navbarMobile in multiple component instead of in the app .jsx :)

**Comment by Romain-Constant** [comment]
> the major problem I see is that you import navbar / navbarMobile in multiple component instead of in the app .jsx :)

Done !

**Review by fantinerudent — APPROVED** [review]
the major problem  I  see is that you import navbar / navbarMobile in multiple component instead of in the app .jsx :)

**Pull request merged**